packages/cysync-core/src/bgTask/networkTask/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+
+import logger from '~/utils/logger';
+
+import { setNetworkState, useAppDispatch } from '../..';
+
+/**
+ * ***************************** WARNING *****************************
+ * To be only used via `Context`. Only 1 instance of useNetwokStatus
+ * should be active in the whole application.
+ */
```

Comment:
We can remove this since this is not a context

Reply:
removed